Dockerfile-cpu

```diff
@@ -1,12 +1,35 @@
 FROM pytorch/torchserve:latest-cpu
+
 USER root
+
 RUN pip install "onnxruntime==1.14.1" "onnx==1.13.1" "git+https://github.com/facebookresearch/segment-anything.git"
+
 COPY ./deployment/dockerd-entrypoint.sh /usr/local/bin/dockerd-entrypoint.sh
+
 RUN chmod +x /usr/local/bin/dockerd-entrypoint.sh
+
 RUN mkdir -p /home/model-server/ && mkdir -p /home/model-server/tmp
+
 COPY ./deployment/config_decode.properties /home/model-server/config.properties
+
+COPY . ./
+
+RUN mkdir model-weights/ \
+    && curl -o model-weights/sam_vit_h_4b8939.pth https://dl.fbaipublicfiles.com/segment_anything/sam_vit_b_01ec64.pth \
+    && torch-model-archiver --model-name sam_vit_h_encode --version 1.0.0 --serialized-file model-weights/sam_vit_h_4b8939.pth --handler handler_encode.py
+
+RUN python scripts/export_onnx_model.py --checkpoint model-weights/sam_vit_h_4b8939.pth --model-type vit_h --output sam_vit_h_decode.onnx
+
+RUN torch-model-archiver --model-name sam_vit_h_decode --version 1.0.0 --serialized-file sam_vit_h_decode.onnx --handler handler_decode.py
+
+RUN mv *.onnx model-store/
```

Comment:
If folks are locally testing, this would move all onyx models into the model_store and potentially balloon the docker image size. also I think we need to move the .mar archives produced from torch-model-archiver in the step before, not the onyx files use to make the .mar.